doc/security.md

```diff
@@ -74,8 +75,9 @@ public class Application {
 As you see there are two arguments passed to the @EnableCustomSecurity:
 
 - authenticator: it introduces a class should be used for authentication.
-- ignoredPaths: using this parameter, you can specify the paths under which the authentication/authorization checks
-    are disabled.
+- applicationBasePathPattern: it is a required parameter that specifies the application's context path pattern.
+- ignoredPaths: note that you do not need to force authentication on all REST paths. By using this parameter, you can
```

Comment:
change description:
excludes some paths to be ignored from base path. By default no path is excluded.